Mathlib/AlgebraicTopology/FundamentalGroupoid/Basic.lean

```diff
@@ -268,33 +268,34 @@ end Path
 
 /-- The fundamental groupoid of a space `X` is defined to be a type synonym for `X`, and we
 subsequently put a `CategoryTheory.Groupoid` structure on it. -/
-def FundamentalGroupoid (X : Type u) := X
+@[ext]
+structure FundamentalGroupoid (X : Type u) where
+  as : X
```

Comment:
I suspect the linter will ask for this anyway, but; can you add a docstring to `as`, preferably one that makes clear the intended mnemonic?